janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/schema/util/GrpcUtils.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.grpc.schema.util;
+
+import com.google.protobuf.Int64Value;
+import org.janusgraph.core.Multiplicity;
+import org.janusgraph.graphdb.grpc.types.EdgeLabel;
+import org.janusgraph.graphdb.grpc.types.VertexLabel;
+
+public class GrpcUtils {
+
+    public static VertexLabel createVertexLabelProto(org.janusgraph.core.VertexLabel vertexLabel) {
+        return VertexLabel.newBuilder()
+            .setId(Int64Value.of(vertexLabel.longId()))
```

Comment:
We are not coming into this part (create label gRPC API) yet, but just leaving a message here to remind us:

One thing we need to check is whether `longId()` returns a permanent id. For user-created vertices, calling `longId()` or `id()` before committing the transaction could lead to an intermediate result, if `ids.flush=false`. Not sure about vertex labels and edge labels though.